lib/wisper/activerecord/publisher.rb

```diff
@@ -15,9 +16,21 @@ module Publisher
 
         # NOTE: do not need to silence deprecations on Rails 5+
         ActiveSupport::Deprecation.silence do
-          after_commit :broadcast_create, on: :create
-          after_commit :broadcast_update, on: :update
-          after_commit :broadcast_destroy, on: :destroy
+          after_commit :broadcast_create, on: :create, if: -> { should_broadcast?(:create) }
+          after_commit :broadcast_update, on: :update, if: -> { should_broadcast?(:update) }
+          after_commit :broadcast_destroy, on: :destroy, if: -> { should_broadcast?(:destroy) }
+        end
+
+        class_attribute :wisper_activerecord_publisher_broadcast_events
+
+        def self.broadcast_on(*events)
+          return self.wisper_activerecord_publisher_broadcast_events = [] if events == [:none]
```

Comment:
Is there another approach to effectively disabling the publisher without using `:none` as a "magic parameter" or use a constant to make it explicit?

Reply:
@wireframe take a look at https://github.com/betterup/wisper-activerecord-publisher/pull/5/commits/1549537c8723af35ab071181f183a3b2f43497e4 and let me know what you think.